tests/test_tools.yml

```diff
@@ -273,34 +273,15 @@
     - path: results/variant_calling/haplotypecaller/test1/test1.haplotypecaller.filtered.vcf.gz
     - path: results/variant_calling/haplotypecaller/test1/test1.haplotypecaller.filtered.vcf.gz.tbi
 
-# - name: Run joint germline variant calling with haplotypecaller
-#   command: nextflow run main.nf -profile test,tools_germline,docker --tools haplotypecaller --joint_germline
-#   tags:
-#     - germline
-#     - gvcf
-#     - haplotypecaller
-#     - variant_calling
-#   files:
-#     - path: results/variant_calling/haplotypecaller/sample1/sample1.haplotypecaller.g.vcf.gz
-#     - path: results/variant_calling/haplotypecaller/sample1/sample1.haplotypecaller.g.vcf.gz.tbi
-#     - path: results/variant_calling/haplotypecaller/sample1/sample1.haplotypecaller.vcf.gz
-#     - path: results/variant_calling/haplotypecaller/sample1/sample1.haplotypecaller.vcf.gz.tbi
-#     - path: results/csv/variantcalled.csv
-
-# - name: Run joint germline variant calling with haplotypecaller without intervals
-#   command: nextflow run main.nf -profile test,tools_germline,docker --tools haplotypecaller --joint_germline --no_intervals
-#   tags:
-#     - germline
-#     - gvcf
-#     - haplotypecaller
-#     - no_intervals
-#     - variant_calling
-#   files:
-#     - path: results/variant_calling/haplotypecaller/sample1/sample1.haplotypecaller.g.vcf.gz
-#     - path: results/variant_calling/haplotypecaller/sample1/sample1.haplotypecaller.g.vcf.gz.tbi
-#     - path: results/variant_calling/haplotypecaller/sample1/sample1.haplotypecaller.vcf.gz
-#     - path: results/variant_calling/haplotypecaller/sample1/sample1.haplotypecaller.vcf.gz.tbi
-#     - path: results/csv/variantcalled.csv
```

Comment:
these tests should still be run

Reply:
which tests? the joint_germline with intervals is done, the joint_germline without intervals does not run. So there's nothing to test

Reply:
Oh, sorry, these were comments...